courses/machine_learning/deepdive2/supplemental/solutions/question_answering_rnn.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<ul><li>variable names: EM, F1 (no upper case)</li><li>pep8</li><li>Is there a way to break that function into smaller functions and give them informative names to understand what happens?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Broke it into two functions, let me know if better!</p>